projects/swimlane/ngx-datatable/src/lib/services/column-changes.service.ts

```diff
@@ -8,9 +8,9 @@ import { Observable, Subject } from 'rxjs';
  */
 @Injectable()
 export class ColumnChangesService {
-  private columnInputChanges = new Subject<undefined>();
+  private columnInputChanges = new Subject<void>();
```

Comment:
This "fix" popped up when using RxJS v7, which is technically correct, although the previous code was working fine. Unfortunately RxJS v6 was not throwing this type issue.